src/sidecar/connections/watcherUtils.test.ts

```diff
@@ -66,6 +68,9 @@ describe("connectionEventHandler", () => {
         reactToCCloudAuthStateStub.calledWith(TEST_CCLOUD_CONNECTION),
         `reactToCCloudAuthState called with ${reactToCCloudAuthStateStub.getCall(0).args[0]}`,
       );
+
+      // ccloud events should never fire directConnectionCreatedStub
+      assert.strictEqual(directConnectionCreatedStub.notCalled, true);
```

Comment:
Minor suggestion for readability:
```suggestion
      sinon.assert.notCalled(directConnectionCreatedStub);
```